app/templating/summary/formatter_registry.py

```diff
@@ -0,0 +1,28 @@
+from app.templating.summary.formatters.check_box import CheckBoxFormatter
+from app.templating.summary.formatters.currency import CurrencyFormatter
+from app.templating.summary.formatters.date_range import DateRangeFormatter
+from app.templating.summary.formatters.radio_button import RadioButtonFormatter
+
+
+KNOWN_FORMATTERS = {
+    "GENERAL": {
```

Comment:
These formatters all appear to apply to single answer fields, whilst daterange is comprised of multiple. Would it be better to name these keys 'SINGLE"/"MULTIPLE" or similar and change the subkey "DATE" to "DATERANGE"? 'GENERAL' could cover everything.